### DISCUSSION TITLE
Configuration Settings in the current Production deployment differ from your current Project Settings.

### DISCUSSION THREAD

**Discussion by smartystreets-frontend** [vercel/community#1364] [Help]
### Summary

I changed a build configuration in the "General" area of my Vercel project and now I'm getting this error message: "Configuration Settings in the current Production deployment differ from your current Project Settings.", but there aren't any prompts on how to fix it.  I see the new dropdown list for "Production Overrides" and a Build Command text input, but I don't know what I should put in there (perhaps `rm -rf *`? just kidding - I wouldn't do that)  

I'm not sure what to do.  Any help would be greatly appreciated.

**Comment by smartystreets-frontend** [comment]
So I just looked at the "Production Overrides" dropdown again and the "Build Command" field is available but I can't enter anything there.  This is mysterious.

**Reply by Natoprog** [reply]
Did you find an answer for you question, how to change/override settings? It is super frustrating for me. I changed build command only for test I couldn't change it again.

**Comment by CaroFG** [comment]
I think you need to redeploy

**Reply by Natoprog** [reply]
I tried to redeploy but it seem to there nothing changed

**Comment by mffrench** [comment]
We have same problem but we're not yet deploying in production and so there is no production deployment. How then could be some "production overrides" ? How can we fix that without deploying in production ? 
 
 Thank you
 
 cc @QuentinGarou

**Comment by phsantiago** [comment]
It's probably your `vercel.json` file.
I was facing the same issue with vercel ignoring some builds, and it was because my vercel.json looked like this:
```json
{
  "buildCommand": "pnpm turbo build",
  "ignoreCommand": "pnpm dlx turbo-ignore"
}
```

**Comment by 0xbuidler** [comment]
Hey, could we have some clarifications on that please @TooTallNate ? 🙏

**Comment by Natoprog** [comment]
I found an answer how you can change it. So basically you have to change your framework preset for whatever you want and click save button. After that you have to get back to the previous framework which you want and click override in row where you want change something and then you can type something. Finally you can click save button. I hope that @vercel will fix it.  I hope I explained it clearly, but if not, feel free to ask.

**Reply by GabenGar** [reply]
Yeah this fixed the issue for me since I was moving my NextJS repo to Turborepo structure. Switching back and forth probably updated commands with turborepo-related changes. Except I didn't have to override anything.

**Comment by jackdishman** [comment]
This happened to me when converting a create react app project to Next.js. 

I fixed this by specifying the framework in the vercel.json which will override the project settings. 

```
{
    "buildCommand": "next build",
    "framework": "nextjs"
  }
```

**Comment by DoubleOTheven** [comment]
You need to remove the `pnpm.lock`, or similar file, and commit it. Vercel will override the build automatically if one of the package lock files are detected

**Comment by oelbaga** [comment]
It's amazing how there's no answer to this yet. We don't have any vercel.json in our repo but still seeing this error when trying to update the "Ignored Build Step" Switching to another framework and back in the general settings didn't change things. I still see this when trying to ignore build unless there are changes in a monorepo:

"Configuration Settings in the current Production deployment differ from your current Project Settings."

**Comment by adi907** [comment]
Facing the same problem. Still not able to find a fix

**Reply by snolan-ethika** [reply]
> Facing the same problem. Still not able to find a fix

* change whatever command you changed back to the original (if possible)
* change your framework preset (e.g. nuxt -> next), click save, then change it back to original, click save
* re-deploy your last deployment...

These steps seemed to have worked for me, not sure which one it actually was, but it's back now, hope that helps...

**Comment by prajjwal295** [comment]
Facing the same issue
